radix_sort.py

```diff
@@ -0,0 +1,78 @@
+from Queue import Queue
```

Comment:
This is almost certainly not the queue you really want to use.  This queue is meant to be a mutli-threading support device to allow passing objects between threads.  You don't really want that, and calling `.get` on an empty queue of this variety will block while the queue waits for more objects to be queued up.

I suspect the class you really want is the `deque` class from the `collections` module.  It's a double-ended queue that can function both as a stack and as a queue, is as fast as you expect, and won't block when empty.